src/components/Button/index.examples.js

```diff
@@ -3,9 +3,78 @@ import {storiesOf} from '@kadira/storybook'
 import {stringFixture, clickHandlerFixture} from '../../utils/Fixtures'
 import Button from '.'
 
+const standardButtonExamples = [
+  {type: 'default'},
+  {type: 'primary'},
+  {type: 'success'},
+  {type: 'warning'},
+  {type: 'danger'}
+]
+
+const dispStyles = 'flex flex-row content-center justify-center flex-wrap items-center vh-100 bg-navy'
+
+// Standard Large Button (Default)
 storiesOf('Button')
-  .addWithInfo('API', () => (
-    <Button onClick={clickHandlerFixture}> 
-      {stringFixture}
-    </Button>
+  .addWithInfo('Large (Default)', `The Large Default Button is available in several different flavors.`, () => (
+    <div className={dispStyles}>
+      {standardButtonExamples.map((btn, i) => {
```

Comment:
This mapping can be replaced with the `.addWithPropsCombinations` plugin as used in the other components; a few examples:
- https://github.com/eggheadio/egghead-io-styleguide/blob/master/src/components/Icon/index.examples.js#L12-L15
- https://github.com/eggheadio/egghead-io-styleguide/blob/master/src/components/Heading/index.examples.js#L14-L17

I would recommend in each `index.examples.js` file just including one base "default" `.addWithInfo` and if the component has variations, also include the `.addWithPropsCombinations` as has been done in the other components ^. This auto renders all possibilities of prop input and creates a prop table for documentation automatically. Make sense?